### ISSUE TITLE
Using menu functionality broken in Node 10.x

### ISSUE THREAD

**Issue by frankawolverton** [losandes/terminal-adventure#1]
Node v10.x appears to have broken the menu functionality.

## Description
A menu loads just fine, but when attempting to use the up or down arrow keys to change the current selection, the app crashes. 

Using the most basic menu example,
```javascript
const adventure = require('terminal-adventure')

adventure.menu('Choose one', ['One', 'Two', 'Three'])
  .then((answer) => {
    console.log('You chose:', answer)
    adventure.complete()
  });
```
The error message given is:
```
readline.js:1024
            throw err;
            ^

Error: write EPIPE
    at afterWriteDispatched (internal/stream_base_commons.js:75:25)
    at writeGeneric (internal/stream_base_commons.js:70:3)
    at ReadStream.Socket._writeGeneric (net.js:760:5)
    at ReadStream.Socket._write (net.js:772:8)
    at doWrite (_stream_writable.js:413:12)
    at writeOrBuffer (_stream_writable.js:397:5)
    at ReadStream.Writable.write (_stream_writable.js:295:11)
    at Object.cursorTo (readline.js:1068:12)
    at Object.removeLines (/Users/Frank/Development/testNode/node_modules/terminal-adventure/src/Writer.js:24:16)
    at clear (/Users/Frank/Development/testNode/node_modules/terminal-adventure/src/Menu.js:117:18)
Emitted 'error' event at:
    at onwriteError (_stream_writable.js:428:12)
    at onwrite (_stream_writable.js:459:5)
    at _destroy (internal/streams/destroy.js:40:7)
    at ReadStream.Socket._destroy (net.js:605:3)
    at ReadStream.destroy (internal/streams/destroy.js:32:8)
    at afterWriteDispatched (internal/stream_base_commons.js:75:17)
    at writeGeneric (internal/stream_base_commons.js:70:3)
    [... lines matching original stack trace ...]
    at doWrite (_stream_writable.js:413:12)

```

Using nvm to switch back to v8.x gets rid of the error.

## Notes
Two interesting pieces of behavior to note.

- Attaching the process to the visual studio debugger stops the error from being thrown. Behavior is consistent between v8.x and v10.x when running the code this way.

- After the app crashes, terminal stops showing inputted text (including not registering enter key presses as new lines), but does receive inputted text since further commands can be executed.

**Comment by frankawolverton** [comment]
Based on some investigation, some others began to run into somewhat similar error messages around 2 weeks ago right after Node v10 was released. I can't tell yet if it is a bug introduced in v10.x or if they made a change to either readline or how they process stdin. 

Either way, not super urgent for us. Figured you might find it interesting though. We'll probably continue to look at the issue casually as we approach upgrading to v10 and if it hasn't been fixed by then, we'll take a look at fixing it and submitting a PR.

**Comment by losandes** [comment]
Thanks for the heads up Frank. Hopefully I'll get to this before you ;)

**Comment by losandes** [comment]
Fixed in [v1.0.2](https://github.com/losandes/terminal-adventure/releases/tag/1.0.2)
